tests/python/gaia-ui-tests/gaiatest/apps/cost_control/regions/ftu_step3.py

```diff
@@ -29,14 +28,21 @@ def __init__(self, marionette):
         Wait(self.marionette).until(lambda m: view.location['x'] == 0)
 
     def enable_data_alert_toggle(self):
```

Comment:
:+1: for removing the `if`, I think we also need to update the name of the function to `toggle_data_alert()` to reflect the change.